### PR TITLE
String aug

### PR DESCRIPTION
**Pull request by christianvedels**
Implemented string augmentation based on the TextAttack library. Whether this is used is controlled by an argument to OCCDataset `new_attack` which defaults to False.

### CODE REVIEW

File: histocc/attacker.py [hunk 1]
```diff
@@ -5,64 +5,121 @@
...
-                    occ_as_word_list.insert(insert_index, random_word)
+                        # insert the word into the list
+                        occ_as_word_list.insert(insert_index, random_word)
 
-                    x_string_copy[i] = " ".join(occ_as_word_list)
+                        x_string_copy[i] = " ".join(occ_as_word_list)
 
         return x_string_copy
+
+if __name__ == '__main__':
```
Comment (TorbenSDJohansen) [inline histocc/attacker.py:117]: While helpful when developing, I think we should remove this `if __name__ == '__main__`. This script should never be used as the entry point, but keeping it signals valid entrypoint.

File: histocc/datasets0.py
Comment (TorbenSDJohansen) [inline histocc/datasets0.py:1]: Why do we rename this file?

Comment (christianvedels) [reply histocc/datasets0.py:1]: This has a naming conflict with TextAttack

Comment (TorbenSDJohansen) [reply histocc/datasets0.py:1]: Ultimately turned out not to lead to naming conflict and change retracted

**Pull request merged**